pkg/controller/configmap/configmap_controller.go

```diff
@@ -756,3 +775,29 @@ func secretNotEqual(s1 *corev1.Secret, s2 *corev1.Secret) bool {
 	}
 	return compareKey(s1, s2, "tls.crt") || compareKey(s1, s2, "tls.key") || compareKey(s1, s2, "tls.ca")
 }
+
+func podSecurityPolicySupport(c *rest.Config) bool {
```

Comment:
Do you think we can check K8s version only once when NCP Operator starts, or you think K8s may do upgrade during operator runtime?

Reply:
Thanks for advice. Putting k8s version check in SharedInfo during startup initialization. So there is only one time effort to check k8s version and record result into a flag.